jiralib.el

```diff
@@ -942,10 +942,10 @@ Return nil if the field is not found"
         return (cdr (assoc 'displayName user))))
 
 (defun jiralib-get-user-account-id (project full-name)
```

Comment:
I didn't test it - the change looks good, but leaves the function name as a misnomer - if it's no longer returning an account-id, it should be changed (or left alone, and a new one added) called `jiralib-get-user-name` or similar.